openvino_tensorflow/kernels/encapsulate_op.cc

```diff
@@ -96,7 +96,7 @@ NGraphEncapsulateOp::NGraphEncapsulateOp(OpKernelConstruction* ctx)
   m_iter = 0;
   // if this is set to 1, run time cost based assignment of clusters will be
   // disabled
-  m_disable_cost_based_assignment = 0;
+  m_disable_cost_based_assignment = 1;
```

Comment:
We need to revisit this and enable by default as disabling cost based fallback would affect performance of most OD models.